CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) 
+(modification: no type change headlines) and this project adheres to 
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [2.2.0] - 2017-08-03
+- Renamed ``root`` functions argument to ``nodeRef`` for passing a node reference
+- Make ``findPath()`` (path to node for given key) a public method
+
+[2.2.0]: https://github.com/ethereumjs/merkle-patricia-tree/compare/v2.1.2...v2.2.0
+
+## [2.1.2] - 2016-03-01
+- Added benchmark (see [./benchmarks/](./benchmarks/))
+- Updated dependencies
+
+[2.1.2]: https://github.com/ethereumjs/merkle-patricia-tree/compare/v2.1.1...v2.1.2
+
+## [2.1.1] - 2016-01-06
+- Added README, API documentation
+- Dependency updates
+
+[2.1.1]: https://github.com/ethereumjs/merkle-patricia-tree/compare/2.0.3...v2.1.1
+
+## [2.0.3] - 2015-09-24
+- Initial, currently released version
```

Comment:
Does the remark "currently released version" belong to the changelog?

Reply:
Not sure what I wanted to express with this, will rebase later and then change this as well. :-)